Dockerfile

```diff
@@ -61,14 +61,14 @@ RUN cd text-generation-webui-tmp && python download-model.py --text-only decapod
 # Get LoRA
 RUN cd text-generation-webui-tmp && python download-model.py samwit/alpaca7b-lora && mv loras/samwit_alpaca7b-lora ../alpaca7b_lora
 
-COPY *.py .
+COPY src .
```

Comment:
I don't think this is quite right. I tried to build the image and run it to test it for you, but the symlinks below were not pointing to anything.

If they were `ln -s ../alpaca_lora_4bit/autograd_4bit.py ./autograd_4bit.py` (remove 'src/') then they would have linked. So I recommend, either change the copy or change the symlinking.

Reply:
whoops, `COPY src .` didn't do what I thought 🤦

Reply:
Dockerfile updated!

Reply:
I won't be able to test that for a bit. I broke my machine pretty badly.